src/await-to-js.ts

```diff
@@ -6,7 +6,7 @@
 export function to<T, U = Error> (
   promise: Promise<T>,
   errorExt?: object
-): Promise<[U | null, T | undefined]> {
+): Promise<[U, undefined]> | Promise<[null, T]> {
```

Comment:
Maybe a bit cleaner:

```suggestion
): Promise<[U, undefined] | [null, T]> {
```